doc/contributing/documentation_guide.md

```diff
@@ -173,14 +173,10 @@ Code that is a simple string should include the quote marks.
 In general, \RDoc's auto-linking should not be suppressed.
 For example, we should write `Array`, not `\Array`.
 
-We might consider whether to suppress when:
+Suppress when:
 
 - The word in question does not refer to a Ruby entity
   (e.g., some uses of _Class_ or _English_).
-- The reference is to the current class document
-  (e.g., _Array_ in the documentation for class `Array`).
-- The same reference is repeated many times
-  (e.g., _RDoc_ on this page).
 - The reference is to a class or module that users
   usually don't deal with, including these:
```

Comment:
I recommend removing the last bullet as well.  It mentions `Class`, `Module`, and `Method`.  First, I'm not sure users don't usually deal with them.  Second, even if they don't usually deal with them, there is no problem auto-linking when referring to the Ruby class/entity.

I think the only time auto-linking should be supressed is if the usage does not refer to the Ruby class/entity.  Maybe:

```
In general, \RDoc's auto-linking should not be suppressed.
For example, we should write `Array`, not `\Array`.
However, suppress when the word in question does not refer to the Ruby entity
(e.g., some uses of _Class_ or _English_).
```

Reply:
Done.